### PR TITLE
Improve contrast

### PR DESCRIPTION
**Pull request by pabo3000**

### CODE REVIEW

File: Products/CMFPlone/static/plone-compiled.js [hunk 1]
```diff
@@ -21708,5 +21708,5 @@ require([
 
 });
 
-define("/Users/nathan/code/coredev5/src/Products.CMFPlone/Products/CMFPlone/static/plone.js", function(){});
+define("/home/pab/workspace/buildout.coredev/src/Products.CMFPlone/Products/CMFPlone/static/plone.js", function(){});
```
Comment (hvelarde) [inline Products/CMFPlone/static/plone-compiled.js]: do we really need to have an absolute reference here?

**Comment by jensens** [comment]
dont forget a line in CHANGELOG.rst

**Comment by pbauer** [comment]
cherry-picked. Thanks!

File: Products/CMFPlone/static/plone-compiled.js (hunk 1 above)
Comment (bloodbare) [inline Products/CMFPlone/static/plone-compiled.js]: solved on https://github.com/plone/Products.CMFPlone/commit/0245527f7efe463db01640e2f48d9370151a966b

**Pull request closed without merging**